### ISSUE TITLE
[REQ] Add new Feature text file with nightly build

### ISSUE THREAD

**Issue by ghost** [sequelpro/sequelpro#276]
Issue imported from Google Code: https://code.google.com/p/sequel-pro/issues/detail?id=276
Reporter:mjpta...@gmail.com Date:2009-05-21 17:27:05 Status:WontFix Closed:2010-07-01

For those of us who brave nightly builds  I would suggest it might help testing
if once a developing feature is ready ie the feature developer believes it works
if they could add a simple line in an accompanying text file
a very brief description eg:

[<a href="/p/sequel-pro/source/detail?r=747">r747</a>] new feature: [issue reference] Copy data with Columns Names 
[<a href="/p/sequel-pro/source/detail?r=654">r654</a>] bug fix [issue reference] an irritating bug

just an idea to help with testing and feedback
Any feedback can be written in the issues forum.

michael
